app/helpers/notifications_helper.rb

```diff
@@ -38,34 +36,34 @@ def notification_header(notification)
     text = if notification.medium?
       medium_notification_card_header(notifiable)
     elsif notification.course? || notification.lecture?
-      t('notifications.course_selection')
+      t("notifications.course_selection")
     elsif notification.lecture_announcement?
       announcement_notification_card_header(notifiable)
     else
-      link_to t('mampf_news.title'), news_path, class: 'text-dark'
+      link_to(t("mampf_news.title"), news_path, class: "text-dark")
     end
-    text.html_safe
+    text.html_safe # rubocop:todo Rails/OutputSafety
   end
```

Comment:
I am not sure what we are suppposed to do here. In three of the cases that happen, `text` will be a `link_to` (`medium_notification_card_header` and `announcement_notification_card_header` return such) which is safe from my understanding, and in the remaining case it is a string from our local file. Maybe we could do this?

```ruby
  def notification_header(notification)
    notifiable = notification.notifiable
    if notification.medium?
      medium_notification_card_header(notifiable)
    elsif notification.course? || notification.lecture?
      t("notifications.course_selection").html_safe
    elsif notification.lecture_announcement?
      announcement_notification_card_header(notifiable)
    else
      link_to(t("mampf_news.title"), news_path, class: "text-dark")
    end
  end
```

There is this [gist](https://gist.github.com/joekur/73779c40c481a2f8a44f) which might be helpful.

Reply:
fixed in b26b11f2